tests/storage/storage.go

```diff
@@ -1164,7 +1173,80 @@ var _ = SIGDescribe("Storage", func() {
 				Expect(runningPod.Spec.Containers[0].VolumeDevices).NotTo(BeEmpty())
 				Expect(runningPod.Spec.Containers[0].VolumeDevices[0].Name).To(Equal("disk0"))
 			})
+		})
 
+		Context("disk shareable tunable", func() {
+			var (
+				dv         *cdiv1.DataVolume
+				vmi1, vmi2 *virtv1.VirtualMachineInstance
+			)
+			BeforeEach(func() {
+				dv = tests.NewRandomDataVolumeWithRegistryImport(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskCirros), util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
+				_, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(util.NamespaceTestDefault).Create(context.Background(), dv, metav1.CreateOptions{})
+				Expect(err).To(BeNil())
+				vmi1 = tests.NewRandomVMIWithDataVolume(dv.Name)
+				vmi2 = tests.NewRandomVMIWithDataVolume(dv.Name)
+			})
+
+			It("should successfully start 2 VMs with a shareable disk", func() {
+				setShareable(vmi1, "disk0")
+				setShareable(vmi2, "disk0")
+
+				By("Starting the VirtualMachineInstances")
+				tests.RunVMIAndExpectLaunchWithDataVolume(vmi1, dv, 90)
+				tests.RunVMIAndExpectLaunchWithDataVolume(vmi2, dv, 90)
+			})
+			It("should fail starting the second VM when the disk is not shareable (default)", func() {
+				By("Starting the VirtualMachineInstances")
+				tests.RunVMIAndExpectLaunchWithDataVolume(vmi1, dv, 90)
+				vmi2 = tests.RunVMIAndExpectScheduling(vmi2, 180)
+				Eventually(func() bool {
+					currentVMI, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Get(vmi2.GetObjectMeta().GetName(), &metav1.GetOptions{})
+					Expect(err).To(BeNil())
+					for _, cond := range currentVMI.Status.Conditions {
+						if strings.Contains(cond.Message, `Failed to get \"write\" lock\nIs another process using the image`) {
+							return true
+
+						}
+					}
+					return false
+				}, 120, 1).Should(BeTrue())
+			})
+		})
+		Context("write and read data from a shared disk", func() {
+			It("should successfully write and read data", func() {
+				vmi1 := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+				vmi2 := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+				diskName := "disk1"
+				tests.CreateBlockVolumePvAndPvc("1Gi")
```

Comment:
Wouldn't this suffer from the same problem described in #6739?

Reply:
definitely, need to change this

Reply:
@maya-r I changed this to use the new function [CreateCephPVC](https://github.com/kubevirt/kubevirt/blob/e622b99f6cfe4cbd659ff408dffb411effa2664c/tests/utils.go#L5254) and it now uses a Ceph PVC